game.py

```diff
@@ -20,7 +20,37 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    game_on = True
```

Comment:
Nice job using a flag (a variable that is used to indicate that a condition has been met) to control your while loop 👍 

```suggestion
    is_game_on = True
```

It's common practice to name a variable that references a boolean with `is` so that other devs know that this is flag variable that can change.